web/package.json

```diff
@@ -9,13 +9,14 @@
 		"dev": "next dev",
 		"build": "next build",
 		"start": "next start",
-		"lint": "eslint --fix --ext .js,.ts,.tsx types/ pages/ components/ stories/ && npm run stylelint",
-    "stylelint": "stylelint **/*.scss",
-    "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build",
-    "build-styles": "cd ./style-definitions && style-dictionary build && ./build.sh && cd -",
-    "test": "jest",
-    "format": "prettier --write  **/*.{js,ts,jsx,tsx,css,md,scss}"
+		"lint": "eslint --fix --ext .js,.ts,.tsx types/ pages/ components/ stories/ && npm run css-lint",
+		"css-lint": "prettier --check \"**/*.scss\" && stylelint **/*.scss",
+		"storybook": "storybook dev -p 6006",
+		"build-storybook": "storybook build",
+		"build-styles": "cd ./style-definitions && style-dictionary build && ./build.sh && cd -",
+		"test": "jest",
+		"format": "prettier --write **/*.{js,ts,jsx,tsx,md} && npm run format-css",
```

Comment:
i didn't include `&& npm run format-css` with this. for some reason it didn't work.